wallet/src/Pos/Wallet/Web/Swagger/Instances/Schema.hs

```diff
@@ -99,6 +101,22 @@ instance ToSchema FileData where
                 ]
             & required .~ [ "fdInputFile", "fdFileName", "fdFileCType", "fdFilePath"]
 
+instance ToSchema CT.NewBatchPayment where
+    declareNamedSchema _ = do
+        cAccountIdSchema <- declareSchemaRef (Proxy @CT.CAccountId)
+        listSchema       <- declareSchemaRef (Proxy @(NonEmpty (CT.CId CT.Addr, Coin)))
+        return $ NamedSchema (Just "NewBatchPayment") $
+            sketchSchema example
+                & properties .~
+                    [ ("npbFrom", cAccountIdSchema)
+                    , ("npbTo", listSchema)
+                    ]
+      where
+        example = CT.NewBatchPayment
+            { CT.npbFrom = CT.CAccountId "<account id>"
```

Comment:
@neongreen If I can get super greedy, I would love to change `<account id>` to `<walletId@accountId>`, as `CAccountId` is indeed a wallet ID and an account ID all bundled into a string, separated by a "@".

Reply:
done